### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by Cliftonz**
Updated all the items on the site.
added new sections
added pictures for officers
added styling for most sections

### CODE REVIEW

**Comment by Cliftonz** [comment]
Checks are failing because /cslabs does not exist in this repository

**Comment by TheOtherUnknown** [comment]
Will the cslabs page be directed by the reverse proxy when this is pushed? I'd rather not have a 404ed page in prod.

File: _includes/getinvolved.html [hunk 1]
```diff
@@ -2,7 +2,15 @@
...
+            <p>Interested in getting involved in the CSG? </p>
+        <p class="flexlabel">
+            <a class="btn btn-primary" href="https://docs.google.com/forms/d/e/1FAIpQLSe4G1Dl1mrnlTnKrIXhlsAmsvUzTmCNywHFqcVDVnXx3fH4Pg/viewform">Fill out our interest form</a> 
+            <a class="btn btn-primary" href="https://list.ius.edu/sympa/subscribe/csg-announce-l">Join our mailing list</a>
+            <a class="btn btn-primary" href="index.html#schedule">Attend an event</a>
+            <a class="btn btn-primary" href="https://discord.gg/XGfzKxA">Join our Discord Server</a>
+            <a class="btn btn-primary" href="https://trello.com/invite/iuscomputerscigroup/9c12b7af83c5e84cef44fdb5d1f2d9e5">Join our Tello boads</a>
+            <a class="btn btn-primary" href="index.html#resources">Read some of your recomenned resources</a>
```
Comment (TheOtherUnknown) [inline _includes/getinvolved.html]: Typo
```suggestion
            <a class="btn btn-primary" href="index.html#resources">Read our recommended resources</a>    
```

**Comment by Cliftonz** [comment]
No, I will not give a 404. That configuration is what we have currently set up in nginx.

**Pull request merged**